src/lib/components/Button.svelte

```diff
@@ -35,7 +31,7 @@
 <style lang="scss">
   @import "$styles/setup";
 
-  .button {
```

Comment:
Now a `<Button>` has global styles which are not limited to be inside the `<Button>`, if you want to do this you need to add something more to these selectors than this. Either a wrapper around it, or `button` + `a` or similar, these are entirely too loose.